Tasks/DockerComposeV0/dockercomposepush.ts

```diff
@@ -1,6 +1,7 @@
 "use strict";
 
 import * as tl from "azure-pipelines-task-lib/task";
+import * as tr from "azure-pipelines-task-lib/toolrunner";
```

Comment:
Is adding toolrunner required for the Node16 upgrade?

Reply:
It is not required, I removed it